src/web/modules/custom/work_bc_quiz/work_bc_quiz.routing.yml

```diff
@@ -7,3 +7,10 @@ work_bc_quiz.clear_old_node_cron:
     _permission: 'access content'
   options:
     no_cache: 'TRUE'
+work_bc_quiz.report_noc2021_validation:
+  path: '/admin/reports/noc2021-validation'
```

Comment:
There's no menu item in the reports menu... not sure if this is intentional.

Reply:
That was something I meant to add, but forgot as I just copy pasted the path into the url.

I can add it to the menu if you'd like.  Although I'm not really sure its necessary.

Reply:
You're right, it's not necessary - it's just for our use. Ok to leave as is.